<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAdd</class>
 <widget class="QDialog" name="DialogAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0">
    <widget class="QLineEdit" name="lineEdit_5"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Departure</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Is a flight transit, please answer 1 if yes, 0 if no</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Seat number</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>The date and time.  Please, enter date in the following form: YYYY-MM-DD HH:MM.</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Is it an international flight, please answer 1 if yes, 0 if no</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="16" column="0">
    <widget class="QLineEdit" name="lineEdit_8"/>
   </item>
   <item row="21" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Arrival</string>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Visa requirements, answer 0 if there are no visa requirements</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLineEdit" name="lineEdit_9"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Duration.  Please, enter date in the following form: HH:MM. Example: 00:45</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Class</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLineEdit" name="lineEdit_7"/>
   </item>
   <item row="20" column="0">
    <widget class="QLineEdit" name="lineEdit_10"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Airline</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="12" column="0">
    <widget class="QLineEdit" name="lineEdit_6"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string> Example: 2021-01-01 00:15</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogAdd</receiver>
   <slot>addButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>648</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addButtonClicked()</slot>
  <slot>text1Changed()</slot>
 </slots>
</ui>
